spec/system/provider_interface/make_offer_spec.rb

```diff
@@ -38,6 +43,86 @@
     then_the_review_page_is_loaded
     and_i_can_confirm_my_answers
 
+    # locations
```

Comment:
Maybe we should move the change link tests to a new system spec as this one has become quite large?

Reply:
Won't it cause exactly the same issue, as the new system spec does more than the make offer one? We added the code here as we wanted it to be clear that the change links are available via the make offer process as well.

I don't know if we are a bit unecessarily overtesting here however, as we are testing the next step behavior in the wizard spec as well, so maybe we can just leave one step and remove the rest?

Reply:
That's a good question, I think if we test the following two scenarios we don't have to cover every change link and just check that the change links are visible instead:
1) Going through all the steps again when clicking a change link eg. provider
2) Going through some steps that skip a step because there is only one option